st2common/st2common/models/api/pack.py

```diff
@@ -137,6 +137,11 @@ class PackAPI(BaseAPI):
                 'description': 'Specification for the system components and packages '
                                'required for the pack.',
                 'default': {}
+            },
+            'path': {
```

Comment:
This would cause issues if content on different servers is stored in a different location, but that's kinda bad idea anyway and probably many other things would break as well.

Reply:
On a related note - I believe we an an existing function which gets pack directory path (it searches all the search paths).

Should we utilize that here as well? Or deprecate it and just always utilize this db model value?

Reply:
I think I looked into this and settled on pack path being part of model made more sense and was cleaner.